### ISSUE TITLE
AI ignoring waypoints when FRIES is enabled/loaded

### ISSUE THREAD

**Issue by BrianPothoven** [acemod/ACE3#3504]
#**Arma 3 Version:** `1.56` 
**CBA Version:** `2.3.1` 
**ACE3 Version:** `3.5.0` 

**Mods:**
- `@CBA_A3`
- `@ace`

**Description:**
- When a vanilla helicopter is loaded with the FRIES system (both when selected in the attributes (3DEN) as with the FRIES module), the AI ignores the set waypoints and just goes straight up into the air.

**Steps to reproduce:**
- Place a Ghosthawk, open attributes and tick FRIES at the bottom or place the FRIES module and link it to the ghosthawk, give the ghosthawk waypoints, place an AI (Player) next to the helicopter. Launch the mission, board the helicopter and see what happens.

**Where did the issue occur?**
- 3DEN Editor (Singleplayer)

**Placed Modules:**
- Equip FRIES

**RPT log file:**
- N/A
- N/A

**Comment by BaerMitUmlaut** [comment]
This is "normal" AI behaviour and cannot be fixed as far as I know. The AI tries to avoid a collision with the FRIES (which has no collision model btw.), which results in it going straight up.

**Comment by BrianPothoven** [comment]
Oke, is this going to fixed, or is this something unfixable?

**Comment by esteldunedain** [comment]
It has been speculated that changing what object type the FRIE inherits from might fix it. 

So it may be fixable, but it's not certain.

**Comment by BrianPothoven** [comment]
Oke, hope to see it fixed :)

**Comment by olanguedoc** [comment]
I want to add.... 

I added a Fries to a Custom Helo. When I link it to the fries module... the chopper goes up indefinately like mention above.... But... I also noticed that if I place a Pawnee and that one of the choppers is linked to the module... both go up. The Pawnee isnt link to the fries module but is AI is still affected. I am not sure the AI is trying to avoid the fries.

Steps.

PLace 2 helos side by side in "flying" mode. Ex. PAwnee  + GhostHawk. 
PLace a soldier and movetocargo in the GHawk.
Link the ghosthawk to the Fries module.

Both Should go up parallel of each other.
